clients/CMakeLists.txt

```diff
@@ -121,8 +121,19 @@ if( BUILD_CLIENTS_BENCHMARKS OR BUILD_CLIENTS_TESTS)
   if (TARGET OpenMP::OpenMP_CXX)
     set( COMMON_LINK_LIBS "OpenMP::OpenMP_CXX")
     if(HIP_PLATFORM STREQUAL amd)
+      if (NOT WIN32)
+        execute_process (
+          COMMAND bash -c "${HIP_CLANG_ROOT}/bin/clang --version | grep -oP '(?<=Target: ).+' | tr -d '\n'"
+          OUTPUT_VARIABLE LLVM_DEFAULT_TARGET_TRIPLE)
+        if(LLVM_DEFAULT_TARGET_TRIPLE)
+          list( APPEND COMMON_LINK_LIBS "-L\"${HIP_CLANG_ROOT}/lib/${LLVM_DEFAULT_TARGET_TRIPLE}\"")
+        endif()
```

Comment:
I assume you have no way to add this into OpenMP::OpenMP_CXX  as that would be cleaner approach.   I thought we were moving to use /bin/amdclang as well.   Otherwise appears fine

Reply:
The ```find_package(OpenMP)``` is actually using FindOpenMP.cmake from the cmake install itself. From what I have seen, it is finding libgomp.so from a system package and not ROCm. Passing a custom ```CMAKE_PREFIX_PATH``` will not work either for this as they turn off these search paths in FindOpenMP.cmake. This is probably why the ```-L /lib``` had to be added in the first place. The openmp install does not have a config file to query either.

As far as using ```amdclang```, I can make that change.